sbin/build.sh

```diff
@@ -279,18 +279,14 @@ configureVersionStringParameter() {
     if isGnuCompatDate; then
       buildTimestamp=$(date --utc +"%Y-%m-%d %H:%M:%S")
     else
-      buildTimestamp=$(date -u -j +"%Y-%m-%d %H:%M:%S") 
+      buildTimestamp=$(date -u +"%Y-%m-%d %H:%M:%S") 
     fi
   fi
   BUILD_CONFIG[BUILD_TIMESTAMP]="${buildTimestamp}"
 
   # Convert ISO-8601 buildTimestamp string to dateSuffix format: %Y%m%d%H%M
-  local dateSuffix
-  if isGnuCompatDate; then
-    dateSuffix=$(date --utc --date="${buildTimestamp}" +"%Y%m%d%H%M")
-  else
-    dateSuffix=$(date -u -j -f "%Y-%m-%d %H:%M:%S" "${buildTimestamp}" +"%Y%m%d%H%M")
-  fi
+  # "%Y-%m-%d %H:%M:%S" to "%Y%m%d%H%M"
+  local dateSuffix=$(echo "${buildTimestamp}" | cut -d":" -f1-2 | tr -d "\-: ")
```

Comment:
Take this or leave it, but putting the `:` first should avoid the need to escape the `-`
```suggestion
  local dateSuffix=$(echo "${buildTimestamp}" | cut -d":" -f1-2 | tr -d ":- ")
```